src/resolvers/space-sx.ts

```diff
@@ -15,13 +16,18 @@ const SUBGRAPH_URLS = [
 ];
 
 async function getSpaceProperty(key: string, url: string, property: 'avatar' | 'cover') {
+  const id = [key];
+  if (!isStarknetAddress(key)) {
+    id.push(getAddress(key));
+  }
+
   const data = await axios({
     url,
     method: 'POST',
     data: {
       query: `
         query {
-          spaces(where: { id_in: ["${key}", "${getAddress(key)}"] }) {
+          spaces(where: { id_in: ["${id.join('", "')}"] }) {
```

Comment:
maybe this is much better
```suggestion
          spaces(where: { id_in: [${id.map(item => `"${item}"`).join(', ')}] }) {
```